### ISSUE TITLE
Installation restarts Computer without permission

### ISSUE THREAD

**Issue by NitroxydeX** [LRFLEW/OpenRCT2Launcher#4]
The new Launcher restarts the PC right after the installation and do not ask the user if he won't to restart the computer directly or later.

**Comment by LRFLEW** [comment]
Peculiar. The change that probably caused this was switching the appveyor builds from VS 2013 to 2015. I'll look into what needs to be changed to prevent this. Thank you for letting me know.

**Comment by LRFLEW** [comment]
What version of Windows are you using? My clean install of Windows 10 in my VM didn't experience this issue.

**Comment by NitroxydeX** [comment]
I'm using Windows 10. A friend had this problem too. Is also on Windows 10.

Some other friend, I think on Windows 7 or 8.1, didnt had this problem.

On a Windows Server 2012 R2 I had not this problem. There was just a missing dll out of Visual Studio 2013.

**Comment by LRFLEW** [comment]
I think I fixed it, but I want to make sure before I close this issue. Can you try [this installer](https://ci.appveyor.com/api/buildjobs/k15w7u1skckh9ug5/artifacts/OpenRCT2Launcher-win.exe) and see if it still can cause a restart? I know it might be difficult to fully verify, as the installer should only cause a restart one time ever on a machine, but if you can test it to some extent, let me know.

**Comment by LRFLEW** [comment]
I've done some testing on my VMs. I _did_ get a message of my installer asking to restart the computer. I'm not sure how exactly that happened, but it did seem to fix this issue. I'm going to close it. If anybody has this issue again, let me know here and I'll reopen the issue.

**Comment by NitroxydeX** [comment]
Tested in a VM with Windows 7 and on my PC. Didn't even get the message to restart the computer. I uninstalled the Runtime and restarted before I tried. Even with the old Installer no restart. Strange. Maybe there is more in the System that would need to be erased. Seems also only a Windows 10 Problem since the VM is a new Installation 

But I had the message on the first time because I could abort the instant restart.
